### ISSUE TITLE
[v13] media-query-no-invalid is not appropriate for Sass

### ISSUE THREAD

**Issue by caugner** [stylelint/stylelint-config-recommended#199]
### What steps are needed to reproduce the bug?

Use `stylelint-config-recommended` v13 in a project that uses Sass and media queries like [this one](https://github.com/mdn/yari/blob/f1f7cb342331edfdf311f8b25ce7a34cbf5bb55e/client/src/app.scss#L134-L136):

```
@media screen and (max-width: $screen-md) {
  overflow: hidden;
}
```

This seems to be because [media-query-no-invalid](https://stylelint.io/user-guide/rules/media-query-no-invalid/) is not appropriate for Sass.

**Note**: As far as I can see, the README doesn't mention that `stylelint-config-recommended` is not compatible with Sass projects.

### What did you expect to happen?

Stylelint should not complain.

### What actually happened?

Stylelint complains:

```
  134:23  ✖  Unexpected invalid media query "(max-width: $screen-md)"  media-query-no-invalid
```

**Comment by ybiquitous** [comment]
@caugner Thanks for using the template. It seems `stylelint-config-recommended-scss` will disable the rule soon:
https://github.com/stylelint-scss/stylelint-config-recommended-scss/commit/c95076bbf5a2a1443872241fa961388d074a23e6

Stylelint's built-in rules don't or won't handle non-standard syntaxes. So, please just turn off a rule you don't need or use the community's shared configs like `stylelint-config-recommended-scss`.

**Comment by caugner** [comment]
TIL about [stylelint-config-recommended-scss](https://www.npmjs.com/package/stylelint-config-recommended-scss), thanks a lot! 🙌
